### PR TITLE
Refactor for changes in containers API

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: podman/libs/containers.py [hunk 1]
```diff
@@ -167,20 +167,6 @@ def restart(self, timeout=25):
             podman.RestartContainer(self._id, timeout)
             return self._refresh(podman)
 
-    def rename(self, target):  # pylint: disable=unused-argument
```
Comment (4383) [inline podman/libs/containers.py]: Why removing these functions?

Comment (jwhonce) [reply podman/libs/containers.py]: They were pulled from the API.

**Comment by jwhonce** [comment]
> I suppose these changes are related to [containers/libpod@1e6413e](https://github.com/containers/libpod/commit/1e6413e3fde448d3b44fc36fd966d8d57af811f5)
> 
> Can you confirm please?

Correct, API results were changed.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**